src/pages/keyword-density/KeywordDensity.vue

```diff
@@ -3,38 +3,60 @@ import { ref } from 'vue';
 import { MaInput, MaButton } from "@mobileaction/action-kit";
 import "@/pages/keyword-generator/KeywordGenerator.vue"
 import router from "@/router/index.js";
+import {textStore} from "@/stores/textStore.js";
+import {storeToRefs} from "pinia";
 
-const text = ref('');
-const totalCharacters = ref(0);
+const store = textStore()
+const text = storeToRefs((store)).userInputText;
+const totalCharacters = ref(text.value.length);
```

Comment:
`totalCharacter` should be computed property.

```suggestion
const totalCharacters = computed(() => textStore.userInputText.length);
```